src/shared/utils.ts

```diff
@@ -0,0 +1,29 @@
+import Amplify from "aws-amplify";
+import { AuthState, AmplifyConfig } from "shared/interfaces/amplify.interface";
+
+export const isAuthenticated = (state: string) => {
```

Comment:
Used to grant access to the rest of the App (maybe we can also use LazyLoading later)